### DISCUSSION TITLE
Why can I use the docker instruction to detect only XML and some configuration files, but not java files

### DISCUSSION THREAD

**Discussion by huo2huo** [JetBrains/Qodana#76] [Q&A] [answered]
Why can I use the docker instruction to detect only XML and some configuration files, but not java files

![image](https://user-images.githubusercontent.com/101409437/157882685-c78b39ca-3595-413e-b3ef-2ba88461be16.png)
![image](https://user-images.githubusercontent.com/101409437/157882707-d6003c59-52eb-416e-8a6a-198dd75384aa.png)
![image](https://user-images.githubusercontent.com/101409437/157882762-106a01f3-ca91-4247-8be9-14f848848139.png)

**Comment by brichbash** [comment, marked as answer]
Hi!

Could you please share `log` folder from `E:/code/resultnew4` here or send it to [qodana-support@jetbrains.com](mailto:qodana-support@jetbrains.com).?

**Reply by huo2huo** [reply]
[resultnew.zip](https://github.com/JetBrains/Qodana/files/8264516/resultnew.zip)

**Reply by huo2huo** [reply]
resultnew.zip is the generated result

**Comment by huo2huo** [comment]
[bm2.zip](https://github.com/JetBrains/Qodana/files/8264708/bm2.zip)
bm2.zip is the project

**Reply by brichbash** [reply]
Does reloading maven (double press `shift`, "Reload All Maven Projects) in the project work in IntellIJ IDEA for you? If it does, do you use maven `settings.xml`? You may need to mount it to the docker container adding `-v <path/to/settings.xml>:/root/.m2/settings.xml` to `docker run`.

**Reply by huo2huo** [reply]
Thanks for your inspiration. I have solved it.There are many models in my project, and the dependencies under my submodules are not downloaded properly.
![lQLPDhs_d4lWvofNA8vNB3qwl5jjL6ppp08COKhNB0BJAA_1914_971](https://user-images.githubusercontent.com/101409437/159009636-7c632928-4c7f-41cc-a681-f8457d73b2c8.png)
